File: 250717/비교 연산/comparison-operator.py
# 입력
num1, num2 = map(int, input().split())
# [조건 1] A가 B보다 같거나 큰가?
if num1 >= num2 :
    print(1)
else :
    print(0)
# [조건 2] A가 B보다 큰가?
if num1 > num2 :
    print(1)
else :
    print(0)
# [조건 3] B가 A보다 크거나 같은가?
if num2 >= num1 :
    print(1)
else :
    print(0)
# [조건 4] B가 A보다 큰가?
if num2 > num1 :
    print(1)
else :
    print(0)
# [조건 5] A와 B가 같은가?
if num1 == num2 :
    print(1)
else :
    print(0)
# [조건 6] A와 B가 다른가?
if num1 != num2 : 
    print(1)
else :
    print(0)

File: 250718/비교 연산/comparison-operator.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();

        // [조건 1] A가 B보다 같거나 큰가?
        if (num1 >= num2) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
        // [조건 2] A가 B보다 큰가?
        if (num1 > num2) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
        // [조건 3] B가 A보다 같거나 큰가?
        if (num2 >= num1) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
        // [조건 4] B보다 A가 큰가?
        if (num2 > num1) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
        // [조건 5] A와 B는 같은가?
        if (num1 == num2) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
        // [조건 6] A와 B는 다른가?
        if (num1 != num2) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
    }
}